libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2222,9 +2222,9 @@ void GCS_MAVLINK::log_vision_position_estimate_data(const uint64_t usec,
                                            (double)x,
                                            (double)y,
                                            (double)z,
-                                           (double)roll,
-                                           (double)pitch,
-                                           (double)yaw);
+                                           (double)(roll * (180 / M_PI)),
```

Comment:
You need to change the units field (and possibly the multiplier field).

We probably also have a convenient radians-to-degrees method...

Reply:
> We probably also have a convenient radians-to-degrees method...

It's conveniently called `degrees()`

Reply:
thanks @WickedShell and @OXINARF . I found both RAD_TO_DEG and degrees() are the same thing.